README.md

```diff
@@ -9,123 +8,118 @@ java_ks
-8. [Release Notes - Notes on the most recent updates to the module](#release-notes)
 
-Overview
---------
+##Overview
 
 The java_ks module uses a combination of keytool and openssl to manage entries in a Java keystore.
 
-Module Description
-------------------
+##Module Description
 
-The java_ks module contains a type called 'java_ks' and a single provider named 'keytool'.  Their purpose is to enable importation of arbitrary, already generated and signed certificates into a java keystore for use by various applications. 
+The java\_ks module contains a type called `java_ks` and a single provider named `keytool`.  Their purpose is to enable importation of arbitrary, already generated and signed certificates into a java keystore for use by various applications. 
 
-Setup
------
-
-**What java_ks affects:**
-
-* keystore repositories
+##Setup
 
 ### Beginning with java_ks
 
-To use the java_ks module's functionality, declare each java_ks resource you need
-
-    java_ks { 'puppetca:truststore':
-      ensure       => latest,
-      certificate  => '/etc/puppet/ssl/certs/ca.pem',
-      target       => '/etc/activemq/broker.ts',
-      password     => 'puppet',
-      trustcacerts => true,
-    }
-    
-    java_ks { 'puppetca:keystore':
-      ensure       => latest,
-      certificate  => '/etc/puppet/ssl/certs/ca.pem',
-      target       => '/etc/activemq/broker.ks',
-      password     => 'puppet',
-      trustcacerts => true,
-    }
-  
-    java_ks { 'broker.example.com:/etc/activemq/broker.ks':
-      ensure      => latest,
-      certificate => '/etc/puppet/ssl/certs/broker.example.com.pe-internal-broker.pem',
-      private_key => '/etc/puppet/ssl/private_keys/broker.example.com.pe-internal-broker.pem',
-      password    => 'puppet',
-    }
-
-Usage
------
-
-### java_ks
-
-This resource manages the entries in a java keystore, and uses composite namevars to accomplish the same alias spread across multiple target keystores.
+To get started with java_ks, declare each `java_ks` resource you need.
 
-**Parameters within java_ks**
+~~~
+java_ks { 'puppetca:truststore':
+  ensure       => latest,
+  certificate  => '/etc/puppet/ssl/certs/ca.pem',
+  target       => '/etc/activemq/broker.ts',
+  password     => 'puppet',
+  trustcacerts => true,
+}
+~~~
 
-#### `certificate`
 
-An already-signed certificate to place in the keystore.  This file must be present on the node before java_ks{} is run.
+##Usage
 
-To have a java application server use a specific certificate for incoming connections, you will need to simultaneously import the private key accompanying the signed certificate you want to use. As long as you provide the path to the key and the certificate, the provider will do the conversion for you.
+You must specify a target in some way. You can specify `target` after the colon in the title or after the target attribute in the resource. If you declare both, it will prefer the attribute.
```

Comment:
This should probably be something like `using the target attribute` not `after the target attribute`